### ISSUE TITLE
Change default volumes for code on docker-compose.yml

### ISSUE THREAD

**Issue by alonsomaniaco** [wodby/docker4drupal#520]
Volume for ./:/var/www/html:cached is used in some services when you run a existing code but I think it would be a better idea to use something like ./src/:/var/www/html:cached.

My idea is change the default directory for an src folder in which you can place your existing code, instead of the root one, because of it has a lot of files that are not important inside docker or for our application. This would allow too to have that folder with any version control system without deleting this repository origin in parent folder.

**Comment by csandanov** [comment]
Feel free to edit `docker-compose.yml` as you want, we don't want to set requirements how to structure your repo, we just describe a simple use case in our quick start guide: download the archive with files from gh releases, put it in your repo (wherever you like). People do fork this repo and use it as the boilerplate but docker4drupal was never intended to be a boilerplate.

**Comment by alonsomaniaco** [comment]
Yes yes, I opened this because of I think it is better to have it by default our code separated from the docker one.
